### DISCUSSION TITLE
New merchandise idea

### DISCUSSION THREAD

**Discussion by 31trainman** [akash-network/community#427] [Ideas]
I thought I'd be a Neato idea if the akash network teamed up with the web 3 brand pudgy penguins. To release a developer pudgy penguin featuring the Akash network logo on its shirt.

Here's a link to the pudgy penguins website https://pudgypenguins.com

**Comment by CaptainSplince** [comment]
Hey there - I don't see a connection between our project and thiers.

**Reply by 31trainman** [reply]
It's a web 3 project just like we are. Plus it'd a cute way to spread awareness of the Akash network.
